src/components/button.js

```diff
@@ -97,8 +97,13 @@ const Button = ({ children, classes, variant, size, type, disabled, ...rest }) =
     endIcon: `${sizes.iconPadding.endIcon[size]} ${classes.endIcon}`,
   }
 
+  const handleClick = e => {
+    e.stopPropagation()
+    rest.onClick()
```

Comment:
I'm getting a fog here. How do we ensure that `rest` has an onClick method here?

Reply:
Sorry, oversight on my part. I'll check to ensure the prop exists before running onClick